Sparkle/SUAutomaticUpdateAlert.m

```diff
@@ -88,4 +89,30 @@ - (NSString *)descriptionText
     }
 }
 
+- (NSTouchBar *)makeTouchBar
+{
+    NSButton *installButton, *laterButton, *cancelButton;
+    for (NSView *control in self.window.contentView.subviews) {
+        if ([control isKindOfClass:[NSButton class]]) {
+            NSButton *button = (NSButton *)control;
+            if (button.action == @selector(installNow:)) {
+                installButton = button;
```

Comment:
Could you use outlets, like this?

https://github.com/sparkle-project/Sparkle/blob/1.17.x/Sparkle/SUUpdateAlert.m#L47,L49

In the 1.17.x branch I've added one for the automatic update alert as well:
https://github.com/sparkle-project/Sparkle/blob/1.17.x/Sparkle/SUUpdateAlert.m#L47,L49

you could add the other two (it's OK if you change only the English xib, I'll update the rest).